packages/generator-react-server/test/app.js

```diff
@@ -129,7 +130,7 @@ function installDeps() {
 			} else {
 				const localDeps = ['react-server-cli', 'react-server']
 					.map(dep => path.resolve(path.join(__dirname, '../..', dep)));
-				cp.exec('npm install ' + localDeps.join(' '), (error) => {
+				cp.exec(shellescape(['npm', 'install'].concat(localDeps)), (error) => {
```

Comment:
when the current path has spaces in it, it screws up the `cp.exec()` call.  Example:

```cp.exec('npm install /my path with spaces/react-server-cli')```

What should happen instead is something like this:

```cp.exec('npm install /my\ path\ with\ spaces/react-server-cli')```

Same thing would happen if there were a `'` in the path...it would mess up the JavaScript string.